styles/card/card.css

```diff
@@ -12,7 +12,7 @@
   width: 16rem;
   padding: 1rem;
   background-color: var(--secondary-color);
-  box-shadow: 3px 3px var(--default);
+  box-shadow: .5rem .5rem var(--default);
```

Comment:
values less than 10 px can be used in pixel units.

Reply:
raghav told me to make uniformity use the same units, I am confused now